ldap/servers/slapd/back-ldbm/db-bdb/bdb_version.c

```diff
@@ -53,7 +53,12 @@ bdb_version_write(struct ldbminfo *li, const char *directory, const char *datave
     /* Open the file */
     if ((prfd = PR_Open(filename, PR_RDWR | PR_CREATE_FILE | PR_TRUNCATE,
                         SLAPD_DEFAULT_FILE_MODE)) == NULL) {
-        slapi_log_err(SLAPI_LOG_ERR, "bdb_version_write",
+        int sev_level = SLAPI_LOG_ERR;
+        if (PR_GetError() == -5950 /* not found */ && strstr("/dev/shm/", filename)) {
+            /* After a server reboot it is expected for this file to not be present */
+            sev_level = SLAPI_LOG_DEBUG;
```

Comment:
As it is a normal event, we can also log it as SLAPI_LOG_INFO and with a less alarming message like "/dev/shm/slapd-foo not present"

Reply:
> As it is a normal event, we can also log it as SLAPI_LOG_INFO and with a less alarming message like "/dev/shm/slapd-foo not present"

So don't log it as a DEBUG message (not logged by default) to logging it every time?  My idea was to not log it by default as it's just more noise that people complain about.

Reply:
Oopsss my bad. I did not realize that. I agree it is better to not log it (unless debug) as it is expected behavior on reboot. Sorry for the noise.

Reply:
No worries, thanks for the review!